src/main/database.ts

```diff
@@ -0,0 +1,51 @@
+import { CGDatabase } from "../renderer/services/db/api";
+import { LevelDbController } from "../renderer/services/db/controller";
+import { config } from "@chainsafe/eth2.0-config/lib/presets/minimal";
+import { IService } from "../renderer/services/interfaces";
+import { CGAccount } from "../renderer/models/account";
+import fs from 'fs'
+
+export class DatabaseHandler implements IService {
+  private database: CGDatabase;
+  private databasePath: string;
+
+  public constructor() {
+    const random = Math.floor(100000 + Math.random() * 900000)
+    this.databasePath = `test-level-${random}.db`
```

Comment:
you can create config object file where it maps env variables and provides default values.
Because we don't wan't random database names in production

Reply:
you can then pass name into setApp and set it there as env in each test